examples/cndp-frr/docker/cnetfwd-graph.jsonc

```diff
@@ -0,0 +1,178 @@
+{
+    // (R) - Required entry
+    // (O) - Optional entry
+    // All descriptions are optional and short form is 'desc'
+    // The order of the entries in this file are handled when it is parsed and the
+    // entries can be in any order.
+
+    // (R) Application information
+    //    name        - (O) the name of the application
+    //    description - (O) the description of the application
+    "application": {
+        "name": "cnet-graph",
```

Comment:
Should we change this to `cndp-frr` or `cnet-frr` or something like it?

Reply:
yes - makes sense